src/modules/Elsa.Workflows.Api/Endpoints/WorkflowInstances/List/Endpoint.cs

```diff
@@ -33,15 +27,15 @@ public override async Task<Response> ExecuteAsync(Request request, CancellationT
         {
             SearchTerm = request.SearchTerm,
             DefinitionId = request.DefinitionId,
-            DefinitionIds = request.DefinitionIds,
+            DefinitionIds = request.DefinitionIds?.Any() == true ? request.DefinitionIds : null,
             Version = request.Version,
             CorrelationId = request.CorrelationId,
             WorkflowStatus = request.Status,
             WorkflowSubStatus = request.SubStatus,
-            WorkflowStatuses = request.Statuses?.Select(Enum.Parse<WorkflowStatus>).ToList(),
-            WorkflowSubStatuses = request.SubStatuses?.Select(Enum.Parse<WorkflowSubStatus>).ToList(),
-            ToCreatedAt = request.ToCreatedAt,
-            FromCreatedAt = request.FromCreatedAt,
+            WorkflowStatuses = request.Statuses?.Any() == true ? request.Statuses : null,
+            WorkflowSubStatuses = request.SubStatuses?.Any() == true ? request.SubStatuses : null,
+            HasIncidents = request.HasIncidents,
+            TimestampFilters = request.TimestampFilters?.Any() == true ? request.TimestampFilters : null,
```

Comment:
request.TimestampFilters?.Any() is enough not need to request.TimestampFilters?.Any() == true

Reply:
I wish you were right, but unfortunately that doesn't work on nullable booleans:

<img width="1244" alt="image" src="https://github.com/elsa-workflows/elsa-core/assets/938393/d296ac68-4faf-427b-8914-772abb7ed78e">